_layouts/blog.html

```diff
@@ -37,16 +37,18 @@ <h1><a href="/blog/">Trino blog</a></h1>
   </div>
 
   <!-- Comment to remove TOC from a page -->
-  {% if content contains '<!-- No TOC -->' %}
+  {% if content contains '<!-- No TOC -->' or page.show_pagenav==false %}
     {{ content }}
   {% else %}
     <div class="row" style="margin-left:0; margin-right:0;">
-      <div class="col-md-8 col-sm-12">
+      <div class="col-md-3">
+      </div>
+      <div class="col-md-6 col-sm-12">
         <div class="mx-2">
           {% include anchor_headings.html html=content anchorBody="#" %}
         </div>
       </div>
-      <div class="col-md-4 d-md-block d-none">
+      <div class="col-md-3 d-md-block d-none">
```

Comment:
This doesn't look good on narrower screens if we get even one level of nesting:
<img width="774" alt="image" src="https://user-images.githubusercontent.com/104291808/232562860-33801372-7c02-4527-9214-eec93dc0579c.png">

If we want it to look better on larger screens, we should specify large column widths, not make it worse for medium-sized screens.